### ISSUE TITLE
PSR-4 Update 

### ISSUE THREAD

**Issue by a-bashtannik** [nodes-php/nemid#31]
Hi

The package needs to be updated with PSR-4 compliance, Composer 1.x warns about the issue, and Composer 2.0 was already released. I can update the package, but it should be a new major release. I already have a fork with some issues fixed. Are you interested in a pull request or you consider the package abandoned? Thanks!

```
Deprecation Notice: Class Nodes\NemId\PidCprMatch\Settings located in ./vendor/nodes/nemid/src/Webservice/Settings.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Nodes\NemId\PidCprMatch\Responses\Response located in ./vendor/nodes/nemid/src/Webservice/PidCprMatch/Responses/Response.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Nodes\NemId\PidCprMatch\PidCprMatch located in ./vendor/nodes/nemid/src/Webservice/PidCprMatch/PidCprMatch.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by a-bashtannik** [comment]
cc @Casperhr

**Comment by Casperhr** [comment]
Hi @a-bashtannik 

Thanks a lot, please PR it. We are happy to merge it in and release a major release

@steffendsommer

**Comment by rasmusebbesen** [comment]
Fixed in #34
